uber/models.py

```diff
@@ -473,6 +473,7 @@ class Attendee(MagModel, TakesPaymentMixin):
     paid             = Column(Choice(PAYMENT_OPTS), default=NOT_PAID)
     overridden_price = Column(Integer, nullable=True)
     amount_paid      = Column(Integer, default=0)
+    donation_tier    = Column(Choice(ADMIN_DONATION_OPTS), default=NO_DONATION)
```

Comment:
This was a hack to get around not having free donation levels. I'd prefer to implement it properly.